meilisearch-release-changelog/v1.1.0.md

```diff
@@ -0,0 +1,178 @@
+
+Done by @gmourier and @irevoire in #3405 and #3529.
+
+## Lifting the limits on index size and count
+
+Before v1.0, the maximum size of an index on disk was limited according to the `--max-index-size` CLI parameter, and the number of indexes that could be accessed by one Meilisearch process would depend on this value (typically, about 1000 indexes with the default setting of 100GiB for `--max-index-size`).
+
+v1.0 removed `--max-index-size` and fixed the maximum size of indexes to 500GiB, with about 200 concurrent indexes accessible to a Meilisearch process.
+
+The removal of `--max-index-size` was made in preparation for this release: you can now use an unlimited number of indexes, whose maximum size will be only dictated by the memory address space devoted to a single process by the OS. The limit is at about 80TiB under Linux, you might run out of disk space before reaching it!
+
+Done by @dureuill in #3319, #3331 and #3544.
+
+## Experimental feature: monitore the Meilisearch usage by using Prometheus
+
+This release introduces an experimental feature to follow your metrics by using Prometheus.
+
+When launching Meilisearch, use the following flag:
+
+```bash
+meilisearch --experimental-enable-metrics
+```
+
+The `/metrics` route will be available and compatible with Prometheus.
+
+🗣️ Since this feature is experimental, we need you to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/meilisearch/discussions/3518).
+
+⚠️ The experimental feature API can break between 2 versions of Meilisearch. Be careful if using it in production.
+
+Done by @irevoire, @dureuill, and @james-2001 in #3496, #3524, #3538 and #3543.
+
+## Improve the language support
+
+* Enhance Greek support by normalizing diacritics and final sigma
+* Enhance Arabic support by ignoring Tatweel
+* Enhance language detection: the language is now detected when indexing to prevent from wrong language detections during the search
+
+Done in #3347 and on the [Charabia side](https://github.com/meilisearch/charabia/releases/tag/v0.7.1) by @choznerol, @cymruu, @james-2001 and @ManyTheFish.
+
+## Other improvements
+
+* Allow using wildcards (`*`) at the end of index names when creating API keys or tenant tokens (#3174) @Kerollmops & @akhildevelops
+* Customize the CSV delimiter (default: `,`) by using the `csvDelimiter` parameter during the document addition and update (#3505, #3534) @MixusMinimax and @irevoire
+* Improve error messages: introduction of "did you mean ...?" suggestions when doing typos on a Meilisearch parameter (#3492) @irevoire
+* Batch together addition and deletion tasks to speed up indextation. You don't have to do anything to activate it, just enjoy! (#3470) @irevoire
```

Comment:
```suggestion
* Batch addition and deletion tasks together to speed up indexing (#3470) @irevoire
```

Reply:
I added `Automatically` at the beginning of the sentence to make people understand there is nothing to do or even understand -> it just happens by default